src/python/test/behave_tests/features/dx.feature

```diff
@@ -20,7 +20,6 @@ Feature: 1: dx commands example
   Scenario: 1030_CLI file upload test, project select on login
     Given CLI login as user qa_admin_org1
     When CLI create new project
-    # TODO: remove expect error (by default no)
```

Comment:
expect_error is still there, is it according to plan? :D

Reply:
Yes, to show that both are supported :D But it is not mandatory